site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -307,6 +307,64 @@ a ClusterQueue can borrow up to the sum of nominal quotas from all the
 ClusterQueues in the cohort. So for the yamls listed above, `team-b-cq` can
 borrow `12+9` CPUs.
 
+### LendingLimit
+
+To limit the amount of resources that a ClusterQueue can lend to others,
+you can set the `.spec.resourcesGroup[*].flavors[*].resource[*].lendingLimit`
+[quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/) field.
+
+#### Enabling feature LendingLimit
```

Comment:
```suggestion
```

I don't think this section level would be useful.

Reply:
Yes. I was trying to make it level 4 title, but that doesn't seem to work.
removed.